### PR TITLE
[CS] New round of Code Style fixes for components/

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for Issue code style fixes

### Summary of Changes
- Space before opening parenthesis of function call prohibited
- Space after opening parenthesis of function call prohibited
- Block comment text must start on a new line
- Multi-line function call not indented correctly
- preference blockComment style over multiple single line comments
- arrays should end with a comma

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

Some manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style testing on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to implement fixable code style

### Documentation Changes Required
none

### CODE REVIEW

File: components/com_content/models/articles.php [hunk 1]
```diff
@@ -575,9 +576,12 @@ public function getItems()
 
 			$item->params = clone $this->getState('params');
 
-			/*For blogs, article params override menu item params only if menu param = 'use_article'
-			Otherwise, menu item params control the layout
-			If menu item is 'use_article' and there is no article param, use global*/
+			/**
+			 * For blogs, article params override menu item params only if menu param = 'use_article'
+			 *  Otherwise, menu item params control the layout
```
Comment (Quy) [inline components/com_content/models/articles.php]: Remove extra space

File: components/com_content/models/articles.php [hunk 2]
```diff
@@ -575,9 +576,12 @@ public function getItems()
...
+			 * If menu item is 'use_article' and there is no article param, use global
+			 */
+
```
Comment (Quy) [inline components/com_content/models/articles.php]: Remove blank line

File: components/com_content/views/article/view.html.php [hunk 3]
```diff
@@ -220,8 +222,10 @@ protected function _prepareDocument()
 		$pathway = $app->getPathway();
 		$title   = null;
 
-		// Because the application sets a default page title,
-		// we need to get it from the menu item itself
+		/**
+		.* Because the application sets a default page title,
```
Comment (Quy) [inline components/com_content/views/article/view.html.php]: Remove period

Comment (photodude) [reply components/com_content/views/article/view.html.php]: fixed

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 80e5394b27eed651adec3f8e5da4bb60b71d0177<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19201">issues.joomla.org/tracker/joomla-cms/19201</a>.</sub>

File: components/com_content/models/articles.php (hunk 1 above)
Comment (photodude) [reply components/com_content/models/articles.php]: fixed

File: components/com_content/models/articles.php (hunk 2 above)
Comment (photodude) [reply components/com_content/models/articles.php]: fixed

File: components/com_content/views/article/view.html.php [hunk 4]
```diff
@@ -139,7 +139,8 @@ public function display($tpl = null)
 			return;
 		}
 
-		/* Check for no 'access-view' and empty fulltext,
+		/*
```
Comment (Quy) [inline components/com_content/views/article/view.html.php]: Should it be `/*` or `/**`?

Comment (photodude) [reply components/com_content/views/article/view.html.php]: yes

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b6d46a246e25a487e4b18e00e53dd031cc6b9e62<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19201">issues.joomla.org/tracker/joomla-cms/19201</a>.</sub>

**Comment by csthomas** [comment]
I have tested this item :white_check_mark: successfully on b6d46a246e25a487e4b18e00e53dd031cc6b9e62<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19201">issues.joomla.org/tracker/joomla-cms/19201</a>.</sub>

**Pull request merged**